File: Task/Java/SmallestDigit.java
import java.util.Scanner;

public class  LargestDigit{
	public static void main (String... args){
	
	Scanner scanner = new Scanner(System.in);

	System.out.print("Enter a number: ");
	int numbers = scanner.nextInt();
	int smallest = 0;
	for(int count = 0; numbers > 0; count++){
		int digit = numbers % 10;
		if(count == 0)
			smallest = digit;
		numbers = numbers/10;
		if (digit < smallest)
			smallest = digit;
		count +=1;

	}			
	System.out.println(smallest);	
					
	

	}


}

File: Task/JavaScript/SmallestDigit.js
function findLSmallestDigit (number){
let count = 0
let smallest = 10
	while(number > 0){
		let digit = number % 10
		if(digit < smallest)
			smallest = digit
		number = Math.floor(number/10)
		count += 1
	}
	return smallest

}
console.log(findLSmallestDigit(89735678))